crates/testgen/src/testgen.rs

```diff
@@ -11,19 +11,22 @@ use tokio::sync::{mpsc, oneshot};
 use tracing::{debug, error, info};
 
 #[derive(Clone)]
-pub struct FallbackDecoder {
-    sender: mpsc::Sender<FDRequest>,
-    current_child_pid: Arc<AtomicU32>,
+pub struct Testgen {
+    pub sender: mpsc::Sender<TestgenRequest>,
```

Comment:
Why do these internal fields need to be public for everyone?

Reply:
Changed in bbec0dbf45875089fe67ec1e08faf79ab47aacac